### ISSUE TITLE
Some errors occurs when shutdown or reboot on ARM

### ISSUE THREAD

**Issue by niusmallnan** [rancher/os#2602]
**RancherOS Version: (ros os version)**
All

We can see these logs on ARM:
```
$ reboot
...
...
 reboot:error: EOF
.Error grabbing logs: function not implemented
reboot:error: EOF
```

**Comment by niusmallnan** [comment]
It seems that our aarch64 system-docker doesn't support `system-docker logs -f <container>`
I can get these logs if I add `-f`:
```error from daemon in stream: Error grabbing logs: function not implemented```

**Comment by niusmallnan** [comment]
Finally, I patch our system-docker.
The system-docker 17.06-ros5 can support `logs -f`.

**Comment by kingsd041** [comment]
Fixed in RancherOS v1.5.1-rc1
```
[root@rancher ~]# reboot
[            ] reboot:info: Setting reboot timeout to 60 (rancher.shutdown_timeout set to 60)
....................^[            ] reboot:info: Setting reboot timeout to 60 (rancher.shutdown_timeout set to 60)
.=.[            ] reboot:info: Stopping /docker : 6595097ecf75
...............G..............[            ] reboot:info: Stopping /ntp : 6c1df021d2ba
...A....[            ] reboot:info: Stopping /network : 96c0dc1f8aad
............E...........[            ] reboot:info: Stopping /udev : f7d96d6ef15b
...D...[            ] reboot:info: Stopping /system-cron : 798ae58614c6
...@....[            ] reboot:info: Stopping /syslog : fc6015cd9102
...J...[            ] reboot:info: Console Stopping [/console] : 75bbcd8ef077
.Connection to 192.168.2.172 closed by remote host.
Connection to 192.168.2.172 closed.
```
